src/docs/JobPlanner/references/JobPlannerServiceReference.adoc

```diff
@@ -299,4 +299,144 @@ In such case, an *Association failed* notification will be triggered to users su
+Furthermore, the created authorizations can be modified or deleted.
+
+An authorization gives the specified user(s) some access permission to the specific association(s).
+We provide four access types that can be authorized:
+
+* Read: The user or user group can only read the association(s) and visualize its planning in the gantt chart.
+* Write: The user or user group have, in addition to the read right, the privilege to modify the association(s)' variables, deactivate or reactivate it(them).
+* Admin: The user or user group have, in addition to the write rights, the ability to delete an association(s).
+Moreover, they can create, modify or delete the authorizations of other users for the association(s).
+* No Access: The user or group of users have no access to the association(s).
+This access type is usually used when the admin wants to authorize a user group except for some specific users, or for example authorize access to her/his all associations except for some of them.
+
+To manage the association authorizations, first you need to select one then a shield-like button above the associations view in the Job-Planner calendar associations portal will be activated, as shown in the figure below.
+
+image::../../images/Association-authorisations-button.png[align=center]
+
+Once you click on it, a dedicated authorization view will open.
+The figure below shows an example.
+
+image::../../images/Association-authorisations-view.png[align=center]
+
+The authorization view consists in two parts:
+
+* First, there are three buttons that allow the admin to refresh the authorizations list, add a new authorization and delete an existing one
+* Second, under the buttons, there is the list that shows the authorizations that are assigned to the selected association.
+
+Each authorization has the following properties:
+
+* Association ID
+* Grantee (required): the username or the user group name
+* Grantee Type: Group or user
+* Access Type: read, write, admin or no access
+* Creator: The user who created the authorization
+* Priority: This attribute is only used for access associated with a group. Thus, if the user belongs to 2 groups, each of which has a different type of access, the system will choose the access with the highest priority for the user.
+For an authorization assigned to a user, the value is 0, since it will be ignored when calculating the resulting access type.
+
+To create a new authorization, the admin just needs to click on the "+" button.
+Then the view of creating an authorization (as shown in the figure below) will be popped up.
+The admin needs to fill in the authorization attributes (as presented above), then click the "Add authorization" button.
+The figure below shows an example of an authorization which authorize the user named `Olivia` the `write` access to association `1`.
```

Comment:
The figure below shows an example of an authorization which gives the user named `Olivia` `write` access to association `1`.